src/dev/ci_setup/git_setup.sh

```diff
@@ -81,6 +80,9 @@ function checkout_sibling {
 
     function checkout_clone_target {
       pick_clone_target
+      if [[ $cloneBranch = "master"  && $cloneAuthor = "elastic" ]]; then
+        export TEST_ES_FROM=snapshot
```

Comment:
Do we need to checkout the es branch if we are running from a snapshot? Might be good time savings.

Reply:
yeah i was hoping to do that, we reference a dotfile in the repo for setting up the correct java version.  i'll see if there's someone else we can find it

Reply:
Ah, that's right - that's fine for now.

Reply:
This also is only for master - I am fine with that for now. We can always add 6.x to the list.